src/main/java/main/movie/model/dto/MovieResponseDto.java

```diff
@@ -0,0 +1,30 @@
+package main.movie.model.dto;
+
+public class MovieResponseDto {
```

Comment:
in request dto we should have `title` and `description` fields

and in the response dto - `id`, `title` and `description`